### ISSUE TITLE
[Slack][CreateTrainableEquipment] Form completes successfully but throws slack error

### ISSUE THREAD

**Issue by chungl** [Denhac/denhac-webhooks#69]
The slack modal for creating new trainable equipment throws a generic "something went wrong" error in slack on form submission, even though the task completes successfully. There do not appear to be any error logs written.

**Comment by Jnesselr** [comment]
I don't believe this is an issue anymore after we moved the submission tasks to a queue. Please re-open if we see this issue again.
